src/strings.ts

```diff
@@ -143,3 +143,33 @@ export type FiscalCode = t.TypeOf<typeof FiscalCode>;
 export const OrganizationFiscalCode = PatternString("^[0-9]{11}$");
 
 export type OrganizationFiscalCode = t.TypeOf<typeof OrganizationFiscalCode>;
+
+/**
+ * A valid (international) phone number
+ */
+function isItalianMobilePhoneNumber(v: t.mixed): v is ItalianMobilePhoneNumber {
```

Comment:
I don't think we're restricted to italian mobile numbers... we should check what SPID IdPs give us, I'm not sure there is a standard, in the SAML attributes it says "string", perhaps we should just take whatever it comes from SPID without validating it?

Reply:
ultimately we should align with SIA, you should ask them